pages/resume.tsx

```diff
@@ -1,22 +1,83 @@
 import React, { FC } from 'react';
 import Resume from '../components/resume/resume';
-import jbatorski from '../data/resume/jbatorski.json';
+import { Basics, Job } from '../components/resume/types';
+import resumeJson from '../data/resume/resume.json';
+import { ResumeJson, WorkItem } from '../data/resume/types';
 
 interface ResumePageProps {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    data: any
+    basics: Basics;
+    skills: string[];
+    jobs: Job[];
 }
 
-const ResumePage: FC<ResumePageProps> = ({ data }) => {
-    return <Resume data={data} />;
+const ResumePage: FC<ResumePageProps> = uiModel => {
+    return <Resume {...uiModel} />;
 };
 
 export async function getStaticProps() {
+    const uiModel = transform(resumeJson as ResumeJson);
+
     return {
-        props: {
-            data: jbatorski,
+        props: uiModel,
+    };
+}
+
+export default ResumePage;
+
+function transform({ basics, skills, work }: ResumeJson) {
+    const uiModel = {
+        basics: {
+            name: basics.name,
+            email: basics.email,
+            url: basics.url,
+            summary: basics.summary,
         },
+        skills: skills.flatMap(skillSet => skillSet.keywords),
+        jobs: groupWorkItemsByCompany(work),
     };
+
+    return uiModel;
 }
 
-export default ResumePage;
\ No newline at end of file
+function groupWorkItemsByCompany(work: WorkItem[]) {
+    // Using map since it remembers insertion order
+    // so when converting back to array
+    // we don't need to worry bout sorting by job start/end times
+    const jobs = new Map<string, Job>();
```

Comment:
Added this as an inline comment since it's not obvious that the insertion order is one of the big reasons for using a Map. The quick lookups are a more obvious benefit.